package.json

```diff
@@ -13,6 +13,8 @@
     "lint": "cross-env NODE_ENV=production npm run lint:commit && npm run lint:vue && npm run lint:css",
     "build": "vue-cli-service build",
     "build:cordova": "cross-env PLATFORM=cordova vue-cli-service build",
+    "build:ionic": "cross-env PLATFORM=ionic && ionic build",
+    "build:ionic:android": "cross-env PLATFORM=ionic && ionic build && ionic capacitor add android && ionic capacitor sync",
```

Comment:
Shouldn't we consider adding a command for iOS build?

Reply:
We will change all ionic run/build commands after we replace all the plugins. Right now I just added these two for dev reasons.